src/Http/ViewComposers/StatamicGlobalDataComposer.php

```diff
@@ -31,7 +31,7 @@ public function getGlobals()
     public function compose(View $view) : View
     {
         if (!$this->globals) {
-            $this->globals = optionalDeep((object)$this->getGlobals());
+            $this->globals = Cache::driver('array')->rememberForever('rapidez.statamic.globals', fn() => optionalDeep((object)$this->getGlobals()));
```

Comment:
This is already "cached" by the singleton right?

Reply:
I expected that as well, during profiling it did not appear to be a singleton. Instead calling optionalDeep for every view.

Reply:
Mm, that's weird. With my profiling earlier it worked great: https://github.com/rapidez/statamic/pull/73/files + https://github.com/rapidez/statamic/blob/master/src/RapidezStatamicServiceProvider.php#L158

Reply:
I agree it shouldn't have impact, for now i will remove this.
We may get back to it in the future, but it shouldn't be necessary